aiohttp/http_websocket.py

```diff
@@ -428,25 +496,23 @@ def _feed_data(self, data: bytes) -> None:
                         raise WebSocketError(
                             WSCloseCode.INVALID_TEXT, "Invalid UTF-8 text message"
                         ) from exc
-                    msg = tuple.__new__(
```

Comment:
It would be some much nicer if we could lazy decode these since it results in a lot of cycles that could be avoided